share/spice/uv/uv.css

```diff
@@ -0,0 +1,44 @@
+.uv_indicator {
```

Comment:
Please prepend everything with `.zci--uv` so that it doesn't affect other IAs. :)

```css
.zci--uv .uv_risk_low {
    background-color: #FFFFFF;
}
```